README.md

```diff
@@ -26,52 +42,29 @@ protected function getContainerBaseClass()
     return parent::getContainerBaseClass();
 }
 ```
-And it's done! ApiTestCase is working with the default configuration.
-
-Configuration Reference
------------------------
-
-To customize your test suite configuration you can add a few more options to phpunit.xml:
 
-```xml
-<php>
-    <server name="KERNEL_DIR" value="/path/to/dir/with/kernel" />
-    <server name="KERNEL_CLASS_PATH" value="/path/to/kernel/class" />
-    <server name="EXPECTED_RESPONSE_DIR" value="/path/to/expected/responses/" />
-    <server name="MOCKED_RESPONSE_DIR" value="/path/to/mocked/responses/" />
-    <server name="FIXTURES_DIR" value="/path/to/DataFixtures/ORM/" />
-    <server name="OPEN_ERROR_IN_BROWSER" value="true/false" />
-    <server name="OPEN_BROWSER_COMMAND" value="open %s" />
-    <server name="IS_DOCTRINE_ORM_SUPPORTED" value="true/false" />
-</php>
-```
- * `KERNEL_DIR` variable contains a path to kernel of your project. If not set, WebTestCase will look for AppKernel in the folder where you have your phpunit.xml file.
- * `KERNEL_CLASS_PATH` allows you to specify exactly which class in which folder should be used in order to setup the Kernel. 
- * `EXPECTED_RESPONSE_DIR` and `MOCKED_RESPONSE_DIR` variables contain paths to folders with expected and mocked responses. `EXPECTED_RESPONSE_DIR` is used when API result is compared with existing json file. `MOCKED_RESPONSE_DIR` should contains files with mocked responses from outside API's. Both variable can have same value but we recommend to keep it separated. If these values aren't set, ApiTestCase will try to guess location of responses. It will try to look for the responses in a following folders '../Responses/Expected' and '../Responses/Mocked' relatively located to your controller test class.
- * `FIXTURES_DIR` variable contains a path to folder with your data fixtures. By default if this variable isn't set it will search for `../DataFixtures/ORM/` relatively located to your test class . ApiTestCase throws RunTimeException if folder doesn't exist or there won't be any files to load.
- * `OPEN_ERROR_IN_BROWSER` is a flag which turns on displaying error in a browser window. The default value is false.
- * `OPEN_BROWSER_COMMAND` is a command which will be used to open browser with an exception.
- * `IS_DOCTRINE_ORM_SUPPORTED` is a flag which turns on doctrine support includes handy data fixtures loader and database purger.
+And it's done! ApiTestCase is working with the default configuration.
```

Comment:
I moved it lower in the README, ApiTestCase should work out of the box with Symfony-Standard, which should we test before releasing blog post.